configure

```diff
@@ -3203,12 +3208,12 @@ update_model_makefiles() {
 
     message "Checking model makefiles"
 
+    local model_name model_file mkfile
```

Comment:
See my other comment: This line may be omitted and one could write `local` in front of each variable when it is defined.